### PR TITLE
Document package management limitations

### PR DESCRIPTION
**Pull request by mziolekda**

### CODE REVIEW

**Review by tudor-da — APPROVED** [review]
lgtm

File: docs/2.10.0/docs/upgrade/smart-contract-upgrades.rst [hunk 1]
```diff
@@ -375,6 +375,29 @@ been implemented, but may be implemented in future releases.
 -  Contract keys in upgradable packages can only include types defined
    within the same package, or types from the Daml Standard Library.
 
+There are further limitations with respect to managing the packages on a running ledger
```
Comment (carrielaben-da) [inline docs/2.10.0/docs/upgrade/smart-contract-upgrades.rst:378]:
```suggestion
There are further limitations with respect to managing the packages on a running ledger:
```

File: docs/2.10.0/docs/upgrade/smart-contract-upgrades.rst [hunk 2]
```diff
@@ -375,6 +375,29 @@ been implemented, but may be implemented in future releases.
...
+
+- Once a version of a package is uploaded to the ledger, it cannot be replaced or removed.
+  If a package is uploaded by mistake it can only be overriden by uploading an even newer version.
+
+- As a consequence of the above, if a record is extended by mistake with an optional field,
+  that field will need to be part of all future versions.
```
Comment (carrielaben-da) [inline docs/2.10.0/docs/upgrade/smart-contract-upgrades.rst:384]:
```suggestion
  that field must be part of all future versions.
```

**Review by carrielaben-da — COMMENTED** [review]
Small changes

**Review by carrielaben-da — APPROVED** [review]

**Pull request merged**